charts/gsp-cluster/templates/02-gsp-system/egress-networkpolicy.yaml

```diff
@@ -18,5 +18,16 @@ spec:
       - 2223
       - 2224
       - 2225
+  - action: Deny
+    protocol: TCP
+    source:
+      namespaceSelector: namespace notin (verify-metadata-controller, verify-proxy-node-build, verify-proxy-node-integration, verify-proxy-node-prod)
```

Comment:
Doesn’t sandbox cluster also get bootstrapped from gsp, in which case we’d need the appropriate namespaces added? These would be:
`sandbox-proxy-node-dev`
`sandbox-metadata-controller`